src/pretix/base/models/orders.py

```diff
@@ -325,6 +325,16 @@ def gracefully_delete(self, user=None, auth=None):
     def email_confirm_hash(self):
         return hashlib.sha256(settings.SECRET_KEY.encode() + self.secret.encode()).hexdigest()[:9]
 
+    def get_extended_status_display(self):
+        if self.status == Order.STATUS_PENDING:
+            if self.require_approval:
+                return _("Approval pending")
+            elif self.valid_if_pending:
+                return pgettext_lazy("order state", "Pending (confirmed)")
+        elif self.status == Order.STATUS_PAID and self.count_positions == 0:
+            return _("Canceled (paid fee)")
+        return self.get_status_display()
```

Comment:
this returns a string starting with lower case letter, the texts above are upper case. this looks a bit inconsistent in the excel sheet... we could do .title() here but I'm not sure on the impact on other translations. or just keep it as is and live with the inconsistency

![grafik](https://github.com/pretix/pretix/assets/388142/67307bb4-0590-4dd0-b752-236da35486a6)

Reply:
This annoyed me, too, and since you've also noticed it, I'm just gonna go for real, lower-case strings for consistency. Even more translatable strings, but oh well, here we are